explorer/e2e/anvil/anvil-tabs-buttons.spec.ts

```diff
@@ -5,39 +5,34 @@ import { anvilTabs } from "./anvil-tabs";
 test("Expect clicking the activities tab to go to the correct url and to show all of the relevant columns when selected", async ({
   page,
 }) => {
-  const tab = anvilTabs.activities;
-  await page.goto(anvilTabs.datasets.url);
-  await testTab(page, tab);
+  await page.goto(anvilTabs.activities.url);
+  await testTab(page, anvilTabs.datasets);
```

Comment:
Should this be `anvilTabs.activities` and the test below be `anvilTabs.datasets`? (It looks like the comment and implementation is mixed with the test below.)